src/main/service/sessionManagementService.ts

```diff
@@ -0,0 +1,55 @@
+import config from 'config';
+import {Logger} from '@hmcts/nodejs-logging';
+import process from 'process';
+import moment from 'moment';
+import {checkRoles, verifiedRoles} from '../authentication/authenticationHandler';
+
+const logger = Logger.getLogger('session-management');
+const authenticationConfig = require('../authentication/authentication-config.json');
+const FRONTEND_URL = process.env.FRONTEND_URL || 'https://pip-frontend.staging.platform.hmcts.net';
+const defaultSessionExpiry = 60 * 60 * 1000; // default to 1 hour
+
+export class SessionManagementService {
```

Comment:
Extra logs should be removed hear to prevent spam

Reply:
Agree the logs look redundant. have removed :)